osi_trafficsign.proto

```diff
@@ -2135,7 +2130,7 @@ message TrafficSign
                 // Begin: \c #TYPE_TRAFFIC_CALMED_DISTRICT_BEGIN
                 TYPE_TRAFFIC_CALMED_DISTRICT_END = 78;
```

Comment:
Is there a better translation?

Reply:
Living Street?
https://en.wikipedia.org/wiki/Living_street

Reply:
If it is the official name according to Wikipedai -> Go for it :)